src/managed/CommonManaged.props

```diff
@@ -6,8 +6,14 @@
   <PropertyGroup>
     <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../..'))/</RepoRoot>
     <VersionPrefix>2.0.0</VersionPrefix>
+    <!-- This is consistent with previously packaged assemblies, but it's not clear if it's what we want
+         going forward -->
+    <AssemblyFileVersion>2.0.0</AssemblyFileVersion>
```

Comment:
I think the AssemblyFileVersion should have the full build number in it, right?  That way it changes for every build.

Also, it would be nice if we could use the `$(VersionPrefix)` here, so we don't have to change 2 numbers.